src/nvim/main.c

```diff
@@ -1494,9 +1498,16 @@ static void init_params(mparm_T *paramp, int argc, char **argv)
 /// Initialize global startuptime file if "--startuptime" passed as an argument.
 static void init_startuptime(mparm_T *paramp)
 {
+  bool is_embedded_process = false;
```

Comment:
variables in small scopes really don't need such verbose names.
```suggestion
  bool is_embed = false;
```

Reply:
changed